provider/pkg/provider/helm_release.go

```diff
@@ -432,6 +422,113 @@ func (r *helmReleaseProvider) setDefaults(target resource.PropertyMap) {
 	}
 }
 
+// setComputedInputs augments resource inputs with computed inputs.
+func (r *helmReleaseProvider) setComputedInputs(ctx context.Context, urn resource.URN, new *Release) error {
+
+	// generate the template output
+	manifest, err := r.helmTemplate(ctx, urn, new)
+	if err != nil {
+		return err
+	}
+
+	// compute the checksum of the rendered output, to be able to detect changes
+	h := sha256.New()
+	_, err = h.Write([]byte(manifest))
+	if err != nil {
+		return err
+	}
+	new.Checksum = hex.EncodeToString(h.Sum(nil))
+
+	// compute resource names
+	_, resources, err := convertYAMLManifestToJSON(manifest)
+	if err != nil {
+		return err
+	}
+	new.ResourceNames = resources
+
+	return nil
+}
+
+func (r *helmReleaseProvider) helmTemplate(ctx context.Context, urn resource.URN, newRelease *Release) (string, error) {
+	conf, err := r.getActionConfig(newRelease.Namespace)
+	if err != nil {
+		return "", err
+	}
+	client := action.NewInstall(conf)
+	c, path, err := getChart(&client.ChartPathOptions, conf.RegistryClient, r.settings, newRelease)
+	if err != nil {
+		logger.V(9).Infof("getChart failed: %+v", err)
```

Comment:
nit: prefer `%v` here to be consistent with other error formatting elsewhere, though functionally the same logged outcome.